CHANGELOG.next.asciidoc

```diff
@@ -385,6 +385,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Improve Cisco ASA/FTD parsing of messages - better support for identity FW messages. Change network.bytes, source.bytes, and destination.bytes to long from integer since value can exceed integer capacity.  Add descriptions for various processors for easier pipeline editing in Kibana UI. {pull}23766[23766]
 - Updating Oauth2 flow for m365_defender fileset. {pull}24829[24829]
 - Improve PanOS parsing and ingest pipeline. {issue}22413[22413] {issue}22748[22748] {pull}24799[24799]
+- Fix IPtable Pipeline and Ubiquity dashboard. {issue}24878[24878] {pull}24928[24928]
```

Comment:
Ubiquity should be Ubiquiti (with an i at the end)

Reply:
As someone who uses their products, whoops 🤭

Reply:
> 
> 
> As someone who uses their products, whoops 🤭

Doesn't help that according to spellcheck the "right" way is wrong too :)